src/main/java/io/appform/dropwizard/actors/base/Handler.java

```diff
@@ -82,4 +101,43 @@ public void handleDelivery(String consumerTag, Envelope envelope,
     private MessageMetadata messageProperties(final Envelope envelope) {
         return new MessageMetadata(envelope.isRedeliver());
     }
+
+    public static Span buildChildSpan(AMQP.BasicProperties props, Tracer tracer) {
```

Comment:
create a separate class (TracingHandler) for all tracing related methods

Reply:
make the method args final

Reply:
done